fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanFragmentWithCostTest.java

```diff
@@ -1201,7 +1201,7 @@ public void testRemoveAggFromAggTable() throws Exception {
                 " count(C_NAME)" +
                 " from lineorder_flat_for_mv" +
                 " group by LO_ORDERDATE, LO_ORDERKEY");
-
+        Thread.sleep(3000);
```

Comment:
sleep here is strange

Reply:
just for wait mv create finish, the ut is unstable